src/client/CodeEditor/json1PresenceDisplay.ts

```diff
@@ -12,8 +12,10 @@ const debug = false;
 
 // TODO: Make this a setting in the UI.
 // See https://github.com/vizhub-core/vzcode/issues/739
-const enableAutoFollow = false;
-
+export let enableAutoFollow = false;
+export const toggleAutoFollowButton = () => {
+  enableAutoFollow = !enableAutoFollow;
```

Comment:
This is sort of an antipattern, but I get what is happening here. Perhaps we could replace this with a React Ref, which would be more idiomatic.